app/Http/Controllers/Backend/WebhookController.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use App\Enum\WebhookEvent;
+use App\Exceptions\PermissionException;
+use App\Http\Controllers\Controller;
+use App\Http\Resources\StatusResource;
+use App\Http\Resources\UserNotificationResource;
+use App\Models\OAuthClient;
+use App\Models\Status;
+use App\Models\User;
+use App\Models\Webhook;
+use App\Models\WebhookCreationRequest;
+use Carbon\Carbon;
+use Illuminate\Notifications\DatabaseNotification;
+use Illuminate\Support\Facades\Log;
+use Laravel\Passport\Client;
+use Spatie\WebhookServer\WebhookCall;
+
+abstract class WebhookController extends Controller {
+    public static function index(User $user): object {
+        return $user->webhooks;
+    }
+
+    public static function createWebhook(
+        WebhookCreationRequest $request
```

Comment:
Can the WebhookCreationRequest be deleted afterwards? Just to not pollute the database more than necessary?